packages/amplify_auth_cognito/lib/method_channel_auth_cognito.dart

```diff
@@ -233,6 +233,27 @@ class AmplifyAuthCognitoMethodChannel extends AmplifyAuthCognito {
     return res;
   }
 
+  @override
+  Future<AuthUser> getCurrentUser({AuthUserRequest request}) async {
+    AuthUser res;
+    try {
+      final Map<String, dynamic> data =
+      await _channel.invokeMapMethod<String, dynamic>(
+        'getCurrentUser',
+        <String, dynamic>{
+          'data': request != null ? request.serializeAsMap() : {},
+        },
+      );
+      res = _formatAuthUserResponse(data);
+      return res;
+    } on PlatformException catch(e) {
+      _throwError(e);
+    } on AuthError catch(e) {
+      throw(e);
+    }
+    return res;
```

Comment:
There is already a return above. Is this line getting executed?

Reply:
This is to stop the compiler from complaining about not getting a return of the expected type.